server/lib/models/account.go

```diff
@@ -88,30 +95,30 @@ func calc_sha256(source string) string {
 }
 
 //compute_sign is calculate signature of login request.
-func compute_sign(id, secret string, walltime uint64) string {
-	src := strconv.FormatUint(walltime, 10) + id + secret
+func compute_sign(id uint64, secret string, walltime uint64) string {
+	src := strconv.FormatUint(walltime, 10) + strconv.FormatUint(id, 10) + secret
 	return calc_sha256(src)
 }
 
 //compute_user_secret calculates secret for users
-func compute_user_secret(id, pass string, walltime uint64) string {
-	return calc_sha256(strconv.FormatUint(walltime, 10) + id + pass + SECRET)
+func compute_user_secret(id uint64, pass string, walltime uint64) string {
```

Comment:
__[gometalinter]__
* [golint] don't use underscores in Go names; func compute_user_secret should be computeUserSecret